packages/amplication-git-pull-request-service/src/core/commit/commits.service.ts

```diff
@@ -0,0 +1,189 @@
+import { GithubFactory, GitProvider } from '@amplication/git-service';
+import { Inject } from '@nestjs/common';
+import {
+  AmplicationLogger,
+  AMPLICATION_LOGGER_PROVIDER,
+} from '@amplication/nest-logger-module';
+
+const AMPLICATION = 'amplication';
```

Comment:
What is the purpose of this const? Did you mean to call it `const BRANCH_NAME = 'amplication';`?

Reply:
yes I will update it thank you